compiler/ballerina-lang/src/main/java/io/ballerina/projects/ArtifactType.java

```diff
@@ -0,0 +1,23 @@
+/*
+ *  Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Update the license header

Reply:
Done in [3d6d26f](https://github.com/ballerina-platform/ballerina-lang/pull/42183/commits/3d6d26f6be6a80d0c512a58bb48b110408ad15ea) commit